samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Content/Controls/NavigationBarSamplePage.xaml.cs

```diff
@@ -51,14 +52,14 @@ private void ModalFlyout_Opened(object sender, object e)
 		private void M3ModalFlyout_Opened(object sender, object e)
 		{
 			var flyoutContent = (sender as Flyout)?.Content;
-			var modalFrameM3 = VisualTreeHelperEx.GetFirstDescendant<Frame>(flyoutContent, x => x.Name == "M3ModalFrame");
+			var modalFrameM3 = Uno.Toolkit.Samples.Helpers.VisualTreeHelperEx.GetFirstDescendant<Frame>(flyoutContent, x => x.Name == "M3ModalFrame");
```

Comment:
You don't  need the fully qualified name here as you already fully qualified it in the using :)